ciongke/cmd/hook/githubagent.go

```diff
@@ -249,3 +254,26 @@ func (ga *GitHubAgent) deleteAll(pr github.PullRequest) {
 		ga.DeleteRequests <- kr
 	}
 }
+
+// Uses a cache for members, but ignores it for non-members.
+func (ga *GitHubAgent) isMember(name string) (bool, error) {
+	ga.mut.Lock()
+	if ga.orgMembers == nil {
+		ga.orgMembers = make(map[string]bool)
```

Comment:
Yeah, I guess some people do a `map[string]struct{}` here, but I'm not a big fan of that.